src/System.Runtime.Serialization.Primitives/tests/System.Runtime.Serialization.Primitives.Tests.csproj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <ProjectGuid>{43341081-27A8-4CFB-8B89-C63CE457EAC2}</ProjectGuid>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
this line is not required as no pointers are used here

Reply:
Fixed